src/new-client/src/App/Trades/TradesGrid/TradesGrid.tsx

```diff
@@ -8,98 +9,114 @@ import {
 import { capitalize } from "utils/capitalize"
 import { format as formatDate } from "date-fns"
 import { useTableTrades, sortFieldSelections$, useTableSort } from "../services"
+import { FixedSizeList as VirtualizedTable } from "react-window"
+import { useSelectedHeight } from "components/Resizer"
 
 export type ColField = keyof Trade
 export interface ColConfig {
   field: ColField
   valueFormatter?: (val: unknown) => string
   headerName: string
   numeric?: boolean
+  width: number
 }
 
-type CellConfig = Pick<ColConfig, "numeric">
+type CellConfig = Pick<ColConfig, "numeric" | "width">
 
 export const colConfigs: ColConfig[] = [
   {
     headerName: "Trade ID",
     field: "tradeId",
+    width: 100,
   },
   {
     headerName: "Status",
     field: "status",
     valueFormatter: capitalize,
+    width: 110,
   },
   {
     headerName: "Trade Date",
     field: "tradeDate",
     valueFormatter: (v) => formatDate(v as Date, DATE_FORMAT),
+    width: 130,
   },
   {
     headerName: "Direction",
     field: "direction",
+    width: 110,
   },
   {
     headerName: "CCYCCY",
     field: "symbol",
+    width: 110,
   },
   {
     headerName: "Deal CCY",
     field: "dealtCurrency",
+    width: 90,
   },
   {
     headerName: "Notional",
     field: "notional",
     valueFormatter: (v) => formatAsWholeNumber(v as number),
     numeric: true,
+    width: 110,
   },
   {
     headerName: "Rate",
     field: "spotRate",
     valueFormatter: (v) => formatTo6Digits(v as number),
     numeric: true,
+    width: 100,
   },
   {
     headerName: "Value Date",
     field: "valueDate",
     valueFormatter: (v) => formatDate(v as Date, DATE_FORMAT),
+    width: 120,
   },
   {
     headerName: "Trader",
     field: "traderName",
+    width: 110,
   },
 ]
 
 const TableWrapper = styled.div`
   height: calc(100% - 4.75rem);
   overflow-x: scroll;
-  overflow-y: scroll;
+  overflow-y: hidden;
 `
-const Table = styled.table`
+
+const Table = styled.div`
   background-color: ${({ theme }) => theme.core.lightBackground};
   position: relative;
   width: 100%;
   min-width: 60rem;
   border-collapse: separate;
   border-spacing: 0;
 `
-const TableHead = styled.thead`
-  font-size: 0.675rem;
-  text-transform: uppercase;
-`
-const TableHeadRow = styled.tr`
+
+const TableHeadRow = styled.div`
   vertical-align: center;
```

Comment:
This is dead code now that this is not a table.  I would check which of the other styles are no longer relevant.